probe/run_probe.py

```diff
@@ -6,12 +7,13 @@
 from prometheus_client import start_http_server
 
 from events import post_event
-from globals import dataset_id, event_interval
+from globals import event_interval
 from listener import listen_to_websocket
 from utils import log, print_header
 
 
 def main():
+    dataset_id = os.getenv("DATASET_ID")
```

Comment:
Any specific reason? Remove from `globals.py` then?

Reply:
It's only in use here, so I saw it as an opportunity to eliminate a global variable. It's already removed from `globals.py`.

Reply:
Never mind, nothing to see here ;-)